aws/elasticsearch/main.tf

```
@@ -0,0 +1,130 @@
+terraform {
+  required_version = ">= 0.11.2"
+  backend          "s3"             {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+module "kops_metadata" {
+  source       = "git::https://github.com/cloudposse/terraform-aws-kops-data-iam.git?ref=tags/0.1.0"
+  cluster_name = "${var.cluster_name}"
+}
+
+module "vpc" {
```

```suggestion
module "kops_data_network" {
```

not to confuse with an actual `vpc` module